lib/src/base-styles/_nx-read-only.scss

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../scss-shared/nx-variables';
+ @import '../scss-shared/nx-text-helpers';
+ @import '../scss-shared/nx-container-helpers';
+
+.nx-read-only {
+  @include container-vertical;
+  margin: 0 0 $nx-spacing-xl 0;
+}
+
+.nx-read-only__label {
+  @include bold();
+  margin: $nx-spacing-xl 0 $nx-spacing-md 0;
+
+  &:first-child {
+    margin-top: 0;
+  }
+}
+
+.nx-read-only__data {
+  margin: 0 0 $nx-spacing-xs 0;
+  padding: 0;
```

Comment:
This doesn't seem like it does anything

Reply:
> This doesn't seem like it does anything

https://github.com/sonatype/sonatype-react-shared-components/pull/243/commits/e9eb2959db0de3996115e2a74ef5ac4df1174a58